### PR TITLE
MCB-37

### PR DESCRIPTION
**Pull request by Mohamad-Aboda**

### CODE REVIEW

**Review by OsamaRagab520 — APPROVED** [review]

File: config/settings/base.py [hunk 1]
```diff
@@ -80,6 +80,7 @@
 
 LOCAL_APPS = [
     "medium.users",
+    "medium.profiles",
     # Your stuff: custom apps go here
```
Comment (hazemessam) [inline config/settings/base.py:84]: Solve the conflict in line 83

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "python.linting.pylintEnabled": false,
     "python.linting.flake8Enabled": true,
-    "python.linting.enabled": true
-}
\ No newline at end of file
+    "python.linting.enabled": true,
```
Comment (hazemessam) [inline .vscode/settings.json]: Solve the conflict in line 4

**Review by hazemessam — CHANGES_REQUESTED** [review]
Solve the conflicts then merge.

File: config/settings/base.py (hunk 1 above)
Comment (Mohamad-Aboda) [reply config/settings/base.py:84]: Resolve conflicts button is disabled i think you should approve the pull request first then i solve conflict.

Comment (hazemessam) [reply config/settings/base.py:84]: No, it's a protected branch. I think @OsamaRagab520 can disable this from settings.
You can solve the conflict locally or wait until Osama changes the settings.

image

**Review by hazemessam — APPROVED** [review]

File: config/settings/base.py (hunk 1 above)
Comment (OsamaRagab520) [reply config/settings/base.py:84]: merge master on your branch and solve it locally then push the merging commit 
the pull request will be updated automatically

**Pull request merged**